src/panels/lovelace/components/hui-card-options.js

```diff
@@ -0,0 +1,40 @@
+import "@polymer/paper-button/paper-button.js";
+import "@polymer/paper-icon-button/paper-icon-button.js";
+
+import { html } from "@polymer/polymer/lib/utils/html-tag.js";
+import { PolymerElement } from "@polymer/polymer/polymer-element.js";
+
+import "../../../components/ha-card.js";
+
+export class HuiCardOptions extends PolymerElement {
+  static get template() {
+    return html`
+    <style>
+      .actions {
+        border-top: 1px solid #e8e8e8;
+        padding: 5px 16px;
+        background: var(--paper-card-background-color, white);
+      }
+    </style>
+    <ha-card>
```

Comment:
I would not expect a `<ha-card>`, most cards are already in a card. Just try to get your actions glued to existing cards